src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SupportedSentence.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import java.time.LocalDate
+
+data class SupportedSentence(
```

Comment:
Not sure what the sentenceAtCondition and offenceDateCondition is for? Cant see how it's being used?

Can't this just be an enum? and then when you check if it's a valid SentenceType just check the list of enum values

Reply:
So they're not currently being used. Originally this ticket validated the sentence type and that the sentencedAt and offenceDate are valid for the given sentence type. However the second part was descope and will be covered in another ticket, I didn't want to delete it though because we'll need it later. I'll add a code comment.

I've added an enum for the SentenceType as well.